client/odysseus/message/index.tsx

```diff
@@ -41,7 +44,9 @@ const ChatMessage = ( { message, isLast, messageEndRef }: ChatMessageProps ) =>
 			ref={ isLast ? messageEndRef : null }
 			className={ `chatbox-message ${ isUser ? 'user' : 'wapuu' }` }
 		>
-			{ message.content }
+			<Suspense fallback={ null }>
```

Comment:
With `AsyncLoad` you also don't need to use a suspense boundary.

Reply:
Removed